gems/quilt_rails/test/quilt_rails/performance_measurable_test.rb

```diff
@@ -0,0 +1,401 @@
+require 'test_helper'
+require 'statsd-instrument'
+
+module Quilt
+  class PerformanceReportableTest < Minitest::Test
+    include Quilt::Performance::Reportable
+
+    def setup
+      @params = ActionController::Parameters.new({})
+    end
+
+    def test_process_report_sends_a_distribution_for_events
+      @params = ActionController::Parameters.new({
```

Comment:
thought about making a fancy factory here like in the sewing-kit tests but felt like overkill for this atm